src/performance/Score.cpp

```diff
@@ -47,6 +47,8 @@ void Score::AppendToUpdateBatch(UpdateBatch& batch) const
 		TotalValue(),
 		_scoreId
 	));
+
+	batch.AppendAndCommitNonThreadsafe(StrFormat("INSERT INTO score_process_queue (mode, score_id, status) VALUES ({0}, {1}, 1) ON DUPLICATE KEY UPDATE status = 1", static_cast<int>(_mode), _scoreId));
```

Comment:
What is this?

Reply:
![image](https://user-images.githubusercontent.com/8146687/148818886-15fa1284-5301-4416-86b3-2cfe595d8414.png)

I didn't notice but when I initially pulled, this came along with it.

Reply:
I believe you pulled from a wrong repo, your fork seem to be based from peppy/osu-perf instead of ppy/osu-perf

Reply:
oh my, my apologies, i mustn’t have been thinking straight.